pandas/core/dtypes/missing.py

```diff
@@ -445,7 +445,7 @@ def array_equivalent(left, right, strict_nan=False):
                     return False
             else:
                 try:
-                    if np.any(left_value != right_value):
+                    if np.any(np.asarray(left_value != right_value)):
```

Comment:
why is this needed?

Reply:
Because "reduce" ufuncs are not yet implemented (and `np.any` seems to use `np.logical_or.reduce` ufunc under the hood)

Reply:
can you add a comment to this effect